.gitpod.yml

```
@@ -82,12 +82,12 @@ tasks:
 
   - name: Cypress
     init: |
-      cd mattermost-server/webapp/e2e/cypress
+      cd ../mattermost-server/e2e/cypress
       npm i
 
   - name: Playwright
     init: |
-      cd mattermost-server/webapp/e2e/cypress
+      cd ../mattermost-server/e2e/cypress
```

Looks like I goofed here in two separate ways

```suggestion
      cd ../mattermost-server/e2e/playwright
```